genesis/engine/entities/fem_entity.py

```diff
@@ -750,6 +751,113 @@ def set_muscle_direction(self, muscle_direction):
             muscle_direction=muscle_direction,
         )
 
+    def set_vertex_constraints(
+        self, vertex_indices, target_positions=None, link=None, constraint_type="hard", stiffness=0.0, damping=0.0
```

Comment:
Would make more sense to use a boolean here, `is_hard_constraint=True`

Reply:
What about forcing critical damping ? So that there is only one parameter to set instead of 2?